extra/release_tool.py

```diff
@@ -1912,14 +1913,26 @@ def push_latest_docker_tags(state, tag_avail):
             # repository.
             repo = image.associated_components_of_type("git")[0]
             if tip == "latest":
-                minor_version = "latest"
-            else:
+                new_version = "latest"
+            elif tip.count(".") == 1:
                 if image.is_independent_component():
-                    minor_version = state[repo.git()]["version"][
+                    new_version = state[repo.git()]["version"][
                         0 : state[repo.git()]["version"].rindex(".")
                     ]
                 else:
-                    minor_version = overall_minor_version
+                    new_version = overall_minor_version
+            elif tip.isnumeric():
```

Comment:
I found this particular call confusing, especially in contrast to the previous `elif`. Is it better to use `tip.count(".") == 0`?

Reply:
Yes, I also think it is confusing. What I don't like of `tip.count(".") == 0` (my first approach, actually) is that it would go for any other generic string. But yeah, as we are defining the list in this very function, it is kind of safe so to avoid confusion I'll change it.